<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showimageqt5Dialog</class>
 <widget class="QDialog" name="showimageqt5Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1580</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>清智科技</string>
  </property>
  <property name="windowIcon">
   <iconset resource="showimageqt5.qrc">
    <normaloff>:/images/qzlogo.png</normaloff>:/images/qzlogo.png</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>390</y>
     <width>280</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
    background-color:rgb(170, 255, 127)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="label_setpitch_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>设置俯仰角：</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_sethead_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>设置方向角：</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pichangle_in_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_headangle_in_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="show_deadpoint_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color:rgb(0, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>计算消失点</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>0</y>
     <width>280</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
    background-color:rgb(170, 255, 127)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QPushButton" name="showimage_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
    background-color:rgb(0, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>显示图像</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="deadpoint_position_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>消失点位置:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_deadpoint_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pitch_angle_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>俯仰角:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pitchangle_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_headangle_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="head_angle_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>91</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>方向角:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="X_focal_length">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X焦距设置:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Y_focal_length">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Y焦距设置:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_xfocal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_yfocal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Mylabel" name="label_tag">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>560</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
    background-color:rgb(170, 255, 127)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLineEdit" name="lineEdit_imagesize_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="image_size_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>当前图像尺寸:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>630</y>
     <width>281</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame
{
    background-color:rgb(170, 255, 127)
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1、由消失点计算俯仰角和方向角：填入“X焦距”和“Y焦距”编辑框；点击“显示图像”按钮，选择图片；手动点击图片中消失点位置，“消失点位置”编辑框显示消失点坐标，“俯仰角”和“方向角”编辑框显示计算后的俯仰角和方向角值；“当前图像尺寸”编辑框显示当前图片的尺寸。                                                                  2、由俯仰角和方位角计算消失点位置：在“设置俯仰角”和“设置方向角”编辑框中填写所测量的俯仰角和方位角值，点击“计算消失点”按钮，在图片中自动已红色标记标识出消失点位置。※软件中角度值单位均为deg。  ※俯仰角向上为正。</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Mylabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="showimageqt5.qrc"/>
 </resources>
 <connections/>
</ui>
